R/optimise.R

```diff
@@ -1,23 +1,37 @@
 #' @import dplyr
+#' @import tidyr
 do_optimise <- function(data, budget) {
-  zone <- intervention <- total_costs <- total_cases_averted <- i <- j <- NULL # used by dplyr
-  cost_df <- distinct(data, zone, intervention, total_costs) %>%
+  zone <- intervention <- cost <- cases_averted <- i <- j <- NULL # used by dplyr
+
+  # Due to a limitation in the optimiser it currently requires at least two
```

Comment:
I will contact Dirk to see whether we can change the optimiser (or the way we invoke it) in order to avoid this workaround